database/migrations/2017_12_21_170327_add_google2fa_secret_to_users.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+
+class AddGoogle2faSecretToUsers extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function ($table) {
+            $table->string('google2fa_secret')->nullable();
```

Comment:
Could you put `$table->string('google2fa_secret')->after('remember_token')->nullable();` instead?

That way, the new field won't be at the latest position in the table.